### PR TITLE
Switch TW from openjdk 22 to 23

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Review by dcermak — CHANGES_REQUESTED** [review]
The `openjdk_23` marker is missing in `pyproject.toml`

File: tests/test_openjdk.py [hunk 1]
```diff
@@ -75,7 +75,7 @@
             OPENJDK_21_CONTAINER, "21", marks=OPENJDK_21_CONTAINER.marks
         ),
         pytest.param(
-            OPENJDK_22_CONTAINER, "22", marks=OPENJDK_22_CONTAINER.marks
+            OPENJDK_23_CONTAINER, "23", marks=OPENJDK_23_CONTAINER.marks
```
Comment (alexandrevicenzi) [inline tests/test_openjdk.py:78]: it seems this causes tests to break.

`'openjdk_23' not found in `markers` configuration option`

Comment (dirkmueller) [reply tests/test_openjdk.py:78]: yes, fixed now.

**Comment by dirkmueller** [comment]
lets change this to draft until the tests are passing. Guys, there is no need to request changes on PRs that fail the tests.

**Comment by alexandrevicenzi** [comment]
> lets change this to draft until the tests are passing. Guys, there is no need to request changes on PRs that fail the tests.

Lately, I've been trying to add a comment only, because I noticed that PRs get blocked if you request changes, unless you dismiss the review.

**Review by dcermak — APPROVED** [review]

**Pull request merged**